tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_ignore_time.py

```diff
@@ -61,14 +115,38 @@ def prepare_agent(mock_agent_module):
 
 
 def test_ignore_time(get_configuration, configure_environment, restart_modulesd, prepare_agent):
-    """Check if an alert is not fired during the ignore time  interval.
-
-    Args:
-        get_configuration (fixture): Get configurations from the module.
-        configure_environment (fixture): Configure a custom environment for testing.
-        restart_modulesd (fixture): Reset the logs file and start a new monitor.
-        prepare_agent (fixture):Add a mock agent, add a package to it and insert a vulnerability for that package.
-    """
+    '''
+    description: Check if an alert is not fired during the ignore time interval. To do this, it inserts a custom
+                 vulnerability and vulnerable package, it checks the initial vulnerability alert, advances the time
+                 clock before the set time, and check that the alert has not been generated. Finally, it advances the
+                 time clock just after the set time and checks that the alert has been generated.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_modulesd:
+            type: callable
+            brief: Restart the `wazuh-modulesd` daemon.
+        - prepare_agent:
+            type: fixture
+            brief: Add a mock agent, add a package to it and insert a vulnerability for that package.
+
+    assertions:
+        - Verify that alerts do not appear before ignore time was finished.
+
+    input_description:
+        - Three use cases are found in the test module and include ignore time intervals of 3600s, 60m, and 1h. The file
+          real_nvd_feed.json is used to check for vulnerabilities.
+
+    expected_output:
+        - r''.* is vulnerable to .*'
```

Comment:
```suggestion
        - r''.* is vulnerable to .*'
        - r'.*Sending FIM event: (.+)$'
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/pull/2387/commits/9527e297136bc644dad453afc17a0b80df79d096)